code-challenges.test.js

```diff
@@ -32,19 +32,83 @@ const object2 = { number: 0 }
 const object3 = { number: -7 }
 // Expected output: "-7 is not divisible by three"
 
+// First, I have to start with a describe statement with a string and function 
+// I am going to name the test "divisibleBy" because I really don't have a more creative name that that to be honest
+// In my it statement I'm going to copy the end of the challenge to explain whats going to happen or supposed to happen in this test 
+// Because every number is NOT always divisible by 3 I put 2 expects, one when the number is divisible by 3 and one when the number isn't divisible by 3
+
+
+describe("divisibleBy", () => {
+    it("takes a object as an argument and decides if the number inside it is evenly divisible by three or not", () => {
+      expect(divisibleBy (object1, object2)).toEqual(`15 and 0 are divisibleBy 3`)
+      expect(divisibleBy (object3)).toEqual(`-7 is not divisibleBy 3`)
+    })
+  })
+
+// ReferenceError: num1 is not defined
+
 // b) Create the function that makes the test pass.
+// Now I will be making the function to define divisibleBy 
+// A conditional statement will do the job 
+
+const divisibleBy = () => {
+    if (15 % 3 == 0) {
+      return `15 is divisibleBy 3`
+    } else if (0 % 3 == 0) {
+      return `0 is divisible by 3`
+    }
+    else (-7 % 3 !== 0 )
+    {
+        return `-7 is not divisible by 3`
+    }
+}
+
+  // terminal is giving
+//   expect(received).toEqual(expected) // deep equality
+
+//   Expected: "15 and 0 are divisibleBy 3"
+//   Received: "15 is divisibleBy 3"
+
+//     41 | describe("divisibleBy", () => {
+//     42 |     it("takes a object as an argument and decides if the number inside it is evenly divisible by three or not", () => {
+//   > 43 |       expect(divisibleBy (object1, object2)).toEqual(`15 and 0 are divisibleBy 3`)
+//        |                                              ^
+//     44 |       expect(divisibleBy (object3)).toEqual(`-7 is not divisibleBy 3`)
+//     45 |     })
+//     46 |   })
+// I think I'm close but I am stuck and have nothing else to try in my arsenal. I really have no idea what to do. I was working on this all this afternoon while in class until the end of class, and even now at 8:16pm and I just have no clue what to do in this situation. I really can't figure it out :(
```

Comment:
Some things to help guide you in the right direction would be to really look into what you are passing into the function (arguments) and what you defining that information that is passed into the function (parameter).  There is no parameter set in this function which will make your function not dynamic and will also fail the tests you wrote.